scripts/cli

```diff
@@ -58,4 +58,9 @@ cmd_version () {
   git add yarn.lock
 }
 
+cmd_woke () {
+  command -v woke || yarn fail 'woke not installed, see https://github.com/get-woke/woke#installation'
```

Comment:
Calling `yarn fail` from a bash script seems overkill

```suggestion
  if ! command -v woke; then
    echo 'woke not installed, see https://github.com/get-woke/woke#installation'
    exit 1
  fi
```

Reply:
the fail script is exactly what I want to do though 😕 
would you prefer to call the bash script directly? (without yarn)

Reply:
What's wrong with my suggestion?

Reply:
that should work but we loose a bit of ["display sugar"](https://github.com/DataDog/browser-sdk/blob/master/scripts/fail.sh) and duplicate an existing behavior

Reply:
what about replacing the `fail.sh` script with a `cmd_fail () {}` command in `cli` ?

Reply:
lgtm
and remove `yarn fail` then?

Reply:
Yes

Reply:
In the future, I'd be in favor of moving complex 'script' like `test:compat:*` in `cli`, so we can rename `cmd_fail` to a simple function `fail`. I think it would be less weird :)